packages/mongodb-rag-core/src/contentStore/MongoDbEmbeddedContentStore.test.ts

```diff
@@ -177,6 +159,116 @@ describe("MongoDbEmbeddedContentStore", () => {
       "custom-embedded_content"
     );
   });
+  describe("deleteEmbeddedContent", () => {
+    let page: PersistedPage;
+    let anotherPage: PersistedPage;
+    let embeddedContent;
+    beforeEach(async () => {
+      assert(store);
+
+      page = {
+        action: "created",
+        body: "foo",
+        format: "md",
+        sourceName: "source1",
+        metadata: {
+          tags: [],
+        },
+        updated: new Date(),
+        url: "/x/y/z",
+      };
+
+      anotherPage = {
+        action: "created",
+        body: "bar",
+        format: "md",
+        sourceName: "another-source",
+        metadata: {
+          tags: [],
+        },
+        updated: new Date(),
+        url: "/a/b/c",
+      };
+
+      const pages = [page, anotherPage];
```

Comment:
nit: you can lift this above the beforeEach block to just be `const` declarations